src/libstd/net/tcp.rs

```diff
@@ -154,8 +164,7 @@ impl TcpStream {
     /// # Note
     ///
     /// Some platforms do not provide access to the current timeout.
-    #[unstable(feature = "socket_timeout", reason = "RFC 1047 - recently added",
-               issue = "27773")]
+    #[stable(feature = "socket_timeout", since = "1.4.0")]
```

Comment:
The fact that this never works on Windows suggests that maybe we shouldn't be stabilizing it...